dashboard/lib/services/lesson_import_helper.rb

```diff
@@ -6,6 +6,25 @@ def self.update_lockable_lesson(script_levels, lesson_id)
     [create_activity_with_levels(script_levels, lesson_id, 1)]
   end
 
+  def self.create_lesson_resources(cb_resources, course_version_id)
+    cb_resources.map do |cb_resource|
+      raise unless cb_resource['slug']
+      resource = Resource.find_or_initialize_by(
+        course_version_id: course_version_id,
+        key: cb_resource['slug']
+      )
+      resource.name = cb_resource['name']
+      resource.url = cb_resource['url']
+      resource.type = cb_resource['type']
```

Comment:
Are we ok with allowing CB resource types which are not in this list? https://github.com/code-dot-org/code-dot-org/blob/1117763fbc41e816c9b198d37c7771ae1fd7034a/apps/src/lib/levelbuilder/lesson-editor/AddResourceDialog.jsx#L58-L67 I am assuming yes, so that we can preserve existing lesson plans as closely as possible when we bring them over, especially when we get around to importing older courses.

Reply:
That seems reasonable to me. We don't do any validation on the types on the backend so this shouldn't cause any issues.